docs/source/stats.rst

```diff
@@ -216,6 +216,8 @@ to one way ANOVA, but still in developement
    :toctree: generated/
 
    local_fdr
+   fdrcorrection_twostage
+   NullDistribution
```

Comment:
```
multipletests
fdrcorrection
fdrcorrection_twostage
local_fdr
NullDistribution
```

and delete 
multipletests
fdrcorrection0
on lines 199, 200
those two sandbox imports were backwards compatibility imports and outdated